Bugfix/toastr css updates

therobbiedavis opened the pull request:
Closes #248

therobbiedavis committed:
CSS Updates

- Removed BS4 toastr styles
- Reinstituted default non-BS4 toastr styles
- Centered login (accounting for header)
- Adjusted the carousel section heading font-size.
- Added a small padding (5px) on top of the padding for the nav, so the text isn't so close to the nav.

therobbiedavis committed:
Login refresh & toaster styles

- Added new font for login
- Updated login styles
- Hide nav bar on logout
- show nav bar on login
- Added images for new login
- dark styles for login
- dark styles for toastr

therobbiedavis committed:
minified images

therobbiedavis committed:
sonar bug fix

therobbiedavis committed:
updating style url for minified asset

therobbiedavis committed:
Fixes and code smells

- fix for login bg image showing up elsewhere
- fix for code smells
- added font family to nav bar

majora2007 commented:
# Issues
- I'm seeing the background image flash on series-detail page, despite it not supposed to be showing there.
- The background image is too large, even after minification, it's sitting at 2MB which is causing a slow first paint and a flash on loading of login page. 
- Why are we adding a new font family?

majora2007 commented on src/app/user-login/user-login.component.scss:
```diff
@@ -0,0 +1,68 @@
+
+    &::before {
+        content: "";
+        background-image: url('../../assets/images/alfons-morales-YLSwjSy7stw-unsplash-min.jpg');
```
Rename this file to be something more meaningful at first glance

majora2007 commented on src/app/user-login/user-login.component.scss:
```diff
@@ -0,0 +1,68 @@
+
+        .alt:focus {
+            background-color: #343c59;
+            box-shadow: 0 0 0 0.2rem rgb(68 79 117 / 50%)
```
Please keep semicolons after all CSS styles.

majora2007 commented on src/styles.scss:
```diff
@@ -1,12 +1,13 @@
 @import '~bootstrap/scss/mixins/_breakpoints.scss';
 
-@import '~ngx-toastr/toastr-bs4-alert';
+//@import '~ngx-toastr/toastr-bs4-alert';
```
Remove this dead code

majora2007 committed:
Fixed missing label/input linking

majora2007 committed:
Merge branch 'bugfix/toastr-css-updates' of https://github.com/Kareadita/Kavita-webui into bugfix/toastr-css-updates

therobbiedavis commented:
- Flashing should be fixed.
- I can resize the background image to help the size
- New font family is for "Kavita" branding

therobbiedavis committed:
resized, compressed, and minified bg image

- change opacity to dark mode login

therobbiedavis committed:
Changed Spartan font files to variable weight

therobbiedavis committed:
Change requests

- Added font license
- Renamed image used for login bg
- Fixed path in styles where above file was used
- Removed now unused bs4 toastr style import

majora2007 committed:
Merge branch 'develop' of https://github.com/Kareadita/Kavita-webui into bugfix/toastr-css-updates
